src/gui/properties/peerlistdelegate.h

```diff
@@ -41,7 +41,7 @@ class PeerListDelegate: public QItemDelegate {
 
 public:
   enum PeerListColumns {COUNTRY, IP, PORT, CONNECTION, FLAGS, CLIENT, PROGRESS, DOWN_SPEED, UP_SPEED,
-                        TOT_DOWN, TOT_UP, RELEVANCE, IP_HIDDEN, COL_COUNT};
+                        TOT_DOWN, TOT_UP, RELEVANCE, DONWLOADING_PIECE, IP_HIDDEN, COL_COUNT};
```

Comment:
@evsh You misspelled "downloading" as "donwloading" here and below.  If possible, could you please rename this field to "DOWNLOADING_PIECE" instead?